src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -417,18 +408,18 @@
       "type": "parameter",
       "dataType": "bool",
       "defaultValue": "false",
-      "description": "Whether or not to use Central Package Management - package versions are via central Directory.Packages.props file"
+      "description": "Use Central Package Management where package versions are via central Directory.Packages.props file"
     },
     "vscode": {
-      "displayName": "WebAssembly - Visual Studio Code Debugging",
+      "displayName": "Visual Studio Code Debugging",
       "type": "parameter",
       "dataType": "bool",
       "defaultValue": "true",
-      "description": "Adds the Visual Studio Code Debugging support files for WebAssembly"
+      "description": "Adds the Visual Studio Code Debugging support files for WebAssembly and Skia"
     },
     "wasmPwaManifest": {
-      "displayName": "WebAssembly - PWA Manifest",
-      "description": "Enables the generation of a PWA manifest for WebAssembly",
+      "displayName": "PWA Manifest",
+      "description": "Generates a PWA manifest for WebAssembly",
```

Comment:
Probably better to mimic the terminology we have for the Test projects `Include` instead of `Generates`

Reply:
Updated - good suggestion